vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/node.hpp

```diff
@@ -22,24 +22,25 @@
 
 #include <rclcpp/rclcpp.hpp>
 
-#include <autoware_control_msgs/msg/control_command_stamped.hpp>
+#include <autoware_auto_control_msgs/msg/ackermann_control_command.hpp>
+#include <autoware_auto_vehicle_msgs/msg/steering_report.hpp>
 #include <autoware_debug_msgs/msg/float32_multi_array_stamped.hpp>
 #include <autoware_vehicle_msgs/msg/actuation_command_stamped.hpp>
-#include <autoware_vehicle_msgs/msg/shift.hpp>
-#include <autoware_vehicle_msgs/msg/steering.hpp>
 #include <geometry_msgs/msg/twist_stamped.hpp>
+#include <nav_msgs/msg/odometry.hpp>
 
 #include <memory>
 #include <string>
 #include <vector>
 
 namespace raw_vehicle_cmd_converter
 {
-using autoware_control_msgs::msg::ControlCommandStamped;
+using ControlCommandStamped = autoware_auto_control_msgs::msg::AckermannControlCommand;
```

Comment:
It may cause misunderstanding? Don't you think so.

Reply:
@takayuki5168 
change to using ackermann control command here a23e927